proposal/more-math-functions.md

```diff
@@ -0,0 +1,445 @@
+* Return NaN if [abs](#abs)(`$number`) > 1.
+* Return -0 if `$number` is -0.
+* Return asin(`$number`).
+
+#### `atan()`
+
+```
+atan($number)
+```
+
+* [Verify inputs](#verify-inputs).
+* Return -0 if `$number` is -0.
+* Return -90deg if `$number` is -Infinity.
+* Return 90deg if `$number` is Infinity.
+* Return atan(`$number`).
+
+#### `atan2()`
+
+`atan2($y, $x)` is distinct from `atan($y / $x)` because it preserves the
+quadrant of the point in question. For example, `atan2(1, -1)` corresponds to
+the point (-1, 1), which returns 135deg. In contrast, `atan(1 / -1)` and
+`atan(-1 / 1)` resolve first to `atan(-1)`, so both return -45deg.
+
+```
+atan2($y, $x)
+```
+
+* [Verify inputs](#verify-inputs).
+* If the inputs match one of the following edge cases, return the value for that
+  edge case. Otherwise, return normal atan2($y, $x).
+
+##### Edge cases
+
+<table>
+  <thead>
+    <tr>
+      <td colspan="2"></td>
+      <th colspan="6" style="text-align: center">X</th>
+    </tr>
+    <tr>
+      <td colspan="2"></td>
+      <th>−Infinity</th>
+      <th>-finite</th>
+      <th>-0</th>
+      <th>0</th>
+      <th>finite</th>
+      <th>Infinity</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th rowspan="6">Y</th>
+      <th>−Infinity</th>
+      <td>-135deg</td>
+      <td>-90deg</td>
+      <td>-90deg</td>
+      <td>-90deg</td>
+      <td>-90deg</td>
+      <td>-45deg</td>
+    </tr>
+    <tr>
+      <th>-finite</th>
+      <td>-180deg</td>
+      <td>(normal)</td>
+      <td>-90deg</td>
+      <td>-90deg</td>
+      <td>(normal)</td>
+      <td>-0deg</td>
+    </tr>
+    <tr>
+      <th>-0</th>
+      <td>-180deg</td>
+      <td>-180deg</td>
+      <td>-180deg</td>
+      <td>-0deg</td>
+      <td>-0deg</td>
+      <td>-0deg</td>
+    </tr>
+    <tr>
+      <th>0</th>
+      <td>180deg</td>
+      <td>180deg</td>
+      <td>180deg</td>
+      <td>0deg</td>
+      <td>0deg</td>
+      <td>0deg</td>
+    </tr>
+    <tr>
+      <th>finite</th>
+      <td>180deg</td>
+      <td>(normal)</td>
+      <td>90deg</td>
+      <td>90deg</td>
+      <td>(normal)</td>
+      <td>0deg</td>
+    </tr>
+    <tr>
+      <th>Infinity</th>
+      <td>135deg</td>
+      <td>90deg</td>
+      <td>90deg</td>
+      <td>90deg</td>
+      <td>90deg</td>
+      <td>45deg</td>
+    </tr>
+  </tbody>
+</table>
+
+### Units
+
+#### `compatible()`
+
+```
+compatible($number1, $number2)
+```
+
+This function is also available as a global function named `comparable()`.
+
+#### `is-unitless()`
+
+```
+is-unitless($number)
+```
+
+This function is also available as a global function named `unitless()`.
+
+#### `unit()`
+
+```
+unit($number)
+```
+
+This function is also available as a global function named `unit()`.
+
+### Util
+
+#### `abs()`
+
+```
+abs($number)
+```
+
+This function is also available as a global function named `abs()`.
+
+#### `hypot()`
+
+```
+hypot($numbers...)
+```
+
+* Throw error if any argument is not a length value.
```

Comment:
Define "length value". Does this allow unitless numbers?

Reply:
Done. Linked to the precise CSS definition `length`